### ISSUE TITLE
SQL error during installation from dev-master

### ISSUE THREAD

**Issue by knoepfchen23** [pimcore/pimcore#4438]
There is a SQL error which makes it impossible to install Pimcore from "dev-master".
I think the error came from issue #4375.
The message is:
```bash
Running database setup...

  6/10 [================>-----------]  60%


                                                                                                                        
 [ERROR] The following errors were encountered during installation                                                      
                                                                                                                        

 * An exception occurred while executing 'CREATE TABLE `assets_metadata` (
  `cid` int(11) DEFAULT NULL,
  `name` varchar(190) DEFAULT NULL,
  `language` varchar(10) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL DEFAULT '',
  `type` ENUM('input','textarea','asset','document','object','date','select','checkbox') DEFAULT NULL,
  `data` text,
  PRIMARY KEY (`cid`, `name`, `language`),
	INDEX `name` (`name`),
	INDEX `cid` (`cid`)
) DEFAULT CHARSET=utf8mb4;':

SQLSTATE[42000]: Syntax error or access violation: 1171 All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead
```
The error is in File:
```bash
bundles/InstallBundle/Resources/install.sql
```
I installed pimcore with PHP 7.3 and MySQL latest 5.7.x.

**Comment by brusch** [comment]
@knoepfchen23 Unfortunately I wasn't able to reproduce this problem on our testing and development instances, but I guess this commit should fix it 81a3a6df30fe7293105a5629dd53774ad5273f3c

Thanks for a quick confirmation!

**Comment by knoepfchen23** [comment]
@brusch I installed Pimcore from dev-master successfully after that fix. Thanks very much.
